ingestion/overview.mdx

```diff
@@ -120,6 +120,52 @@ CREATE TABLE t1(
 INSERT INTO t1 SELECT * FROM source_iceberg_t1;
 ```
 
+## File source management
+
+RisingWave supports reading data from file sources including AWS S3, GCS, and Azure Blob Storage.
+
+### Batch reading from file source
+
+To read data in batch from file sources, you need to create a materialized view from the source or create a table with the appropriate connector. You can also directly query the file source. Below are examples using AWS S3.
+
+```sql
+-- Create a source that connects to S3
+CREATE SOURCE s3_source WITH ( connector = 's3', ... );
+
+-- Create a materialized view from the source for batch processing
+CREATE MATERIALIZED VIEW mv AS SELECT * FROM s3_source;
+
+-- Create a table using the S3 connector
+CREATE TABLE s3_table ( ... ) WITH ( connector = 's3', ... );
+
+-- Select from the source directly
+SELECT count(*) from s3_source;
+```
+
+### Data type mapping in Parquet
+
+You can use the table function `file_scan()` to read Parquet files from sources. Below is the data type mapping that shows how RisingWave converts data types from file sources in Parquet format.
+
+| File source type | RisingWave type |
+| :----------- | :-------------- |
+| boolean      | boolean         |
+| integer      | int             |
+| long         | bigint          |
+| float        | real            |
+| double       | double          |
+| string       | varchar         |
+| date         | date            |
+| timestamptz  | timestamptz     |
+| timestamp    | timestamp       |
+| decimal      | decimal         |
+| Int8         | Int16           |
+| UInt8        | Int16           |
+| UInt16       | Int32           |
```

Comment:
I think `Int16` should be changed into `smallint`, `Int32` -> `int`, `Int64` -> `bigint`?
cc @xiangjinwu to confirm.

Reply:
The right column for RisingWave shall use the names `smallint`, `int`, `bigint`, `decimal`, `real`, `double precision`.

The left column for Parquet shall also be consistent. Names shall be `int32` and `int64` rather than `integer` and `long`. (And seems `int16` is missing?)